src/v2/Apps/Artist/Routes/Overview/Utils/hasOverviewContent.ts

```diff
@@ -4,16 +4,19 @@ export const hasOverviewContent = ({
   statuses,
   related,
   biographyBlurb,
+  insights,
 }: Pick<
   ArtistApp_sharedMetadata,
-  "statuses" | "related" | "biographyBlurb"
+  "biographyBlurb" | "insights" | "related" | "statuses"
 >): boolean => {
   const showArtistBio = biographyBlurb && Boolean(biographyBlurb.text)
   const showRelatedCategories = related?.genes?.edges?.length! > 0
+  const showCareerHighlights = insights?.length
 
   return Boolean(
     showArtistBio ||
       showRelatedCategories ||
+      showCareerHighlights ||
```

Comment:
As the artist page changes through time this approach is always gonna be a game of chase. Don't have any better ideas ATM but at some point we should re-review.